services/agent/src/simcore_service_agent/info/_consumer.py

```diff
@@ -0,0 +1,22 @@
+import asyncio
+
+from ._constants import HOST, INFO_REQUEST, PORT
+
+_8_BYTES = 8 * 1024
+
+
+async def _async_request_info() -> str:
+    reader, writer = await asyncio.open_connection(HOST, PORT)
```

Comment:
Coming back to my thought on the Application: If I understand this correctly, what you are trying to do here is analogous to the classic ``/health`` entrypoint, right?

Note how these **entire ``info`` submodule ** can be replace by using a ``FastAPI`` application with a single entrypoint:

```python
@app.get("/heath")
async def healthcheck(request: Request):
   # WARNING: evaluate CPU!
    return f"{request.app.list_running()}\nALL OK" 
```

Reply:
replaced all this with FastAPI